releasenotes/notes/0.13/consistent-gate-naming-efa669c7a8d3a654.yaml

```diff
@@ -0,0 +1,148 @@
+---
+deprecations:
+  - |
+    The naming of gate objects and methods have been updated to be more
```

Comment:
Should we call this out in the prelude?

Reply:
That's a good idea, this is a pretty big change. I'll do that right now

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/4057/commits/31da6c99ba275c899eaf6f8483d5edac2b596efb